canonical-web-and-design/ubuntu.com
# remove .html extension from data-form-location values

## Done

Removed `.html` from `data-form-location` attribute on contact forms

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the homepage, click "Contact us" in the footer of the page, and see that the contact form modal appears. Click around the site, and do the same thing on many pages.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/5735
